jax/experimental/global_device_array.py

```diff
@@ -189,86 +188,84 @@ class GlobalDeviceArray:
   devices, and the data is visible to the current process. Global shards are
   those across all devices (including local devices), and the data isn’t visible
   if the shard is on a non-local device with respect to the current process.
-  Please see the `Shard` class to see what information is stored inside that
+  Please see the ``Shard`` class to see what information is stored inside that
   data structure.
 
   Note: to make pjit output GlobalDeviceArrays, set the environment variable
-  `JAX_PARALLEL_FUNCTIONS_OUTPUT_GDA=true` or add the following to your code:
-  `jax.config.update('jax_parallel_functions_output_gda', True)`
+  ``JAX_PARALLEL_FUNCTIONS_OUTPUT_GDA=true`` or add the following to your code:
+  ``jax.config.update('jax_parallel_functions_output_gda', True)``
+
+  Args:
+    global_shape: The global shape of the array
+    global_mesh: The global mesh representing devices across multiple
+      processes.
+    mesh_axes: A sequence with length less than or equal to the rank of the
+      global array (i.e. the length of the global shape). Each element can be:
+        * An axis name of ``global_mesh``, indicating that the corresponding
+          global array axis is partitioned across the given device axis of
+          ``global_mesh``.
+        * A tuple of axis names of ``global_mesh``. This is like the above option
+          except the global array axis is partitioned across the product of axes
+          named in the tuple.
+        * None indicating that the corresponding global array axis is not
+          partitioned.
+        For more information, please see:
+        https://jax.readthedocs.io/en/latest/jax-101/08-pjit.html#more-information-on-partitionspec
+    device_buffers: DeviceArrays that are on the local devices of
+      ``global_mesh``.
 
   Attributes:
-    shape: The global shape of the array.
-    dtype: dtype of the global array.
-    local_shards: List of `Shard`s on the local devices of the current process.
-      Data is available for all local shards.
-    global_shards: List of all `Shard`s of the global array. Data isn’t
+    shape: Global shape of the array.
+    dtype: Dtype of the global array.
+    local_shards: List of ``Shard``s on the local devices of the current process.
+      Data is materialized for all local shards.
+    global_shards: List of all ``Shard``s of the global array. Data isn’t
       available if a shard is on a non-local device with respect to the current
       process.
+    is_fully_replicated: True, if the full array value is present on all devices
+      of the global mesh.
 
-  Example:
+  Example::
```

Comment:
I think this should be `Examples:`

Reply:
No, it doesn't become a code block if there is no double colon.

Reply:
I think it does if it is `Examples:` rather than `Example:`